app/src/main/res/values-in/strings.xml

```diff
@@ -162,38 +168,38 @@
+    <string name="setting_add_shortcut_summary">Tambahkan pintasan yang menarik di layar utama seandainya setelah menyembunyikan aplikasi nama dan ikon sulit untuk dikenali</string>
     <string name="settings_doh_title">DNS melalui HTTPS</string>
-    <string name="settings_doh_description">Solusi untuk DNS poisoning di beberapa negara</string>
+    <string name="settings_doh_description">Solusi untuk permasalahan DNS poisoning di beberapa negara</string>
 
     <string name="multiuser_mode">Mode multi pengguna</string>
     <string name="settings_owner_only">Pemilik perangkat saja</string>
     <string name="settings_owner_manage">Pemilik perangkat mengelola</string>
     <string name="settings_user_independent">Pengguna independen</string>
     <string name="owner_only_summary">Hanya pemilik yang memiliki akses root</string>
-    <string name="owner_manage_summary">Hanya pemilik yang dapat mengelola akses root dan menerima pesan permintaan</string>
-    <string name="user_indepenent_summary">Setiap pengguna memiliki aturan root mereka sendiri</string>
+    <string name="owner_manage_summary">Hanya pemilik yang dapat mengelola dan menerima pesan permintaan akses root</string>
+    <string name="user_indepenent_summary">Setiap pengguna memiliki aturan root mereka masing-masing</string>
 
-    <string name="mount_namespace_mode">Mode ruang-nama mount</string>
-    <string name="settings_ns_global">Ruang-nama global</string>
-    <string name="settings_ns_requester">Ruang-nama warisan</string>
-    <string name="settings_ns_isolate">Ruang-nama terpisah</string>
-    <string name="global_summary">Semua sesi root menggunakan ruang-nama mount global</string>
-    <string name="requester_summary">Sesi root akan mewarisi ruang-nama peminta mereka</string>
-    <string name="isolate_summary">Setiap sesi root akan memiliki ruang-nama tersendiri</string>
+    <string name="mount_namespace_mode">Mode ruang nama mount</string>
+    <string name="settings_ns_global">Ruang nama global</string>
+    <string name="settings_ns_requester">Ruang nama warisan</string>
+    <string name="settings_ns_isolate">Ruang nama terpisah</string>
+    <string name="global_summary">Semua sesi root menggunakan ruang nama mount global</string>
+    <string name="requester_summary">Sesi root akan mewarisi ruang nama peminta mereka</string>
+    <string name="isolate_summary">Setiap sesi root akan memiliki ruang nama tersendiri</string>
 
     <!--Notifications-->
     <string name="update_channel">Update Magisk</string>
-    <string name="progress_channel">Notifikasi Kemajuan</string>
+    <string name="progress_channel">Kemajuan notifikasi</string>
     <string name="download_complete">Download selesai</string>
-    <string name="download_file_error">Kesalahan saat mendownload file</string>
+    <string name="download_file_error">Error saat mendownload file</string>
```

Comment:
Nope. Either keep it "Kesalahan" or use "Galat", but not reverting it back to English.